lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -649,7 +657,10 @@ def cmd_use(*args)
 
             # Use a module by search index
             index_from_list(@module_search_results, mod_name) do |mod|
-              return false unless mod && mod.respond_to?(:fullname)
+              unless mod && mod.respond_to?(:fullname)
+                print_error("Invalid module index: #{mod_name}")
+                return false
+              end
```

Comment:
### Before
Before when an invalid number was used with the `use` command it didn't return anything
![image](https://user-images.githubusercontent.com/69522014/91313319-26879680-e7ad-11ea-96bd-96a666b5149f.png)

### After
Now when an invalid number is used with the `use` command it will return `[-] Invalid module index: 99999`
![image](https://user-images.githubusercontent.com/69522014/91313090-e2949180-e7ac-11ea-8928-7046cab05802.png)

Reply:
what happens if you do `info` on an invalid number/multiple invalid numbers?

Reply:
@dwelch-r7, you beat me on this: https://github.com/rapid7/metasploit-framework/pull/14053#discussion_r477336085

Reply:
Sorry forgot to add this in! 

When an invalid number is used with the `info` command it will return [-] Invalid module index: 9999
![image](https://user-images.githubusercontent.com/69522014/91319890-d1e81980-e7b4-11ea-8b20-0f6fd7aec5a9.png)

Reply:
Great usability change!